addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/TellstickRuntimeException.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.tellstick.internal;
+
+/**
+ * Runtime exception in tellstick binding.
+ *
+ * @author jarle hjortland
+ */
+public class TellstickRuntimeException extends RuntimeException {
```

Comment:
Maybe leave the implementation detail out of the name and just name it `TellstickException `

Reply:
There's already a: `org.tellstick.device.TellstickException`